website/data/docs-navigation.js

```diff
@@ -177,7 +177,7 @@ export default [
       'version',
       {
         category: 'volume',
-        content: ['deregister', 'detach', 'init', 'register', 'status'],
+        content: ['create', 'delete', 'deregister', 'detach', 'init', 'register', 'status'],
```

Comment:
I'll need to update this file now that https://github.com/hashicorp/nomad/pull/10264 has been merged

Reply:
I'm going to update this in the `csi-create-volume` PR rather than here, once I've rebased it on `main`.